ci/scripts/tag-check.sh

```diff
@@ -3,7 +3,19 @@
 set -eu
 
 if [[ -n "${BUILDKITE_TAG-}" ]]; then
-  echo "$BUILDKITE_TAG"
+  case "${BUILDKITE_TAG-}" in
+    # Git tags that starts like charts go module are not considered release tags that would
+    # trigger operator release process.
+    #
+    # As buildkite has separated configuration "Branch Limiting" which is set to `main` and
+    # `v*` this `case` is a noop.
+    connectors-v* | console-v* | operator-v* | redpanda-v*)
```

Comment:
go won't resolve this format correctly it should be `charts/<chartname>/v*`: https://go.dev/wiki/Modules#is-it-possible-to-add-a-module-to-a-multi-module-repository

I think adding this as a [conditional to the buildkite pipeline](https://buildkite.com/docs/pipelines/configure/conditionals) would be much more discoverable. WDYT?

Reply:
ok I missunderstood https://go.dev/doc/modules/managing-source#multiple-module-source

I will delete tags and re do them.